models/Liquid.js

```diff
@@ -0,0 +1,34 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+class Liquid extends Model {}
+
+Liquid.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: false,
```

Comment:
autoIncrement needs to be true here.

Reply:
@Bungycode You can hit "Resolve conversation" since you addressed the comment and the change was met.